ibtk/include/ibtk/ibtk_utilities.h

```diff
@@ -112,6 +113,29 @@ static const bool ENABLE_TIMERS = true;
 
 namespace IBTK
 {
+template <class T>
+inline std::string
+get_data_time_str(const T data_time, const T current_time, const T new_time)
+{
+    const T half_time = static_cast<T>(0.5) * (current_time + new_time);
```

Comment:
@drwells, should I make this simply
```suggestion
    const T half_time = 0.5 * (current_time + new_time);
```
?

Reply:
Templating on the time type seems like a premature optimization to me - what is the advantage of doing it?

Reply:
Fair enough --- let's just leave it as `double`.